cloud/deploys/dev_azure/bin/setup

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+set -e
```

Comment:
nit: For some reason, the `set -e` wasn't enough to exit from the script when my azure subscription id was wrong.

```Subscription '0f8cb250-b44f-4acd-b24e-2524ef9f85ac' not found. Check the spelling and casing and try again.
Done granting current user access deploy log storage account.
....

AttributeError: 'MaxRetryError' object has no attribute 'lower'
To open an issue, please run: 'az feedback'
Start azure_log::initialize_log_file
Creating deploy log storage account...

```